src/main/java/seedu/cakecollate/model/orderitem/Type.java

```diff
@@ -0,0 +1,54 @@
+package seedu.cakecollate.model.orderitem;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.cakecollate.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the type of cake of an order item.
+ * Guarantees: immutable; is valid as declared in {@link #isValidType(String)}
+ */
+public class Type {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Order Type should only contain alphabets and white spaces, and it should not be blank";
```

Comment:
Is there any reason why it should only contain alphabets? Maybe can relax it a bit for types like 12" chocolate cake or something haha

Reply:
oh yes didn't think about that! Will update it accordingly!